mobile/screens/Analytics/AnalyticsDashboardScreen.tsx

```diff
@@ -17,23 +18,72 @@ export default function AnalyticsDashboardScreen(props: any) {
       10, 20, 40, 30, 50, 60, 10, 90, 100, 250, 400, 600,
     ],
   };
-
+  const [completedUsers, setCompletedUsers] = useState(0);
   useEffect(() => {
     BackHandler.addEventListener("hardwareBackPress", function () {
       props.navigation.navigate(Screens.ADMIN_DASHBOARD_SCREEN);
       return true;
     });
+    setCompletedUsers(fakeData.usersCompletedTraining);
   }, []);
 
   return (
     <BaseOverlay
       header={
         <GenericHeader
-          headerTitle="Analytics"
+          headerTitle='Analytics'
           navigationProp={props.navigation}
         />
       }
-      body={<View style={styles.container}></View>}
+      body={
+        <View style={styles.container}>
+          <View style={{ alignItems: "flex-end", marginBottom: -60 }}>
+            <Text style={styles.title}>Users Who Completed Training</Text>
+            <Text style={styles.subtitle}>800 hours</Text>
+          </View>
+          <VictoryPie
+            name={"Hi"}
+            startAngle={(completedUsers / fakeData.totalUsers) * 360}
+            endAngle={(completedUsers / fakeData.totalUsers) * 360 + 360}
+            animate={{
+              duration: 500,
+            }}
+            data={[
+              { x: "Completed", y: completedUsers },
+              { x: "Not Done", y: fakeData.totalUsers },
+            ]}
+            width={350}
+            height={400}
+            style={{
+              labels: { display: "none" },
+              parent: {
+                marginBottom: -70,
+              },
+            }}
+            colorScale={["#d0ceed", "#403bf6"]}
+          />
+          <VictoryLegend
+            x={110}
+            y={0}
+            height={50}
+            title='Legend'
```

Comment:
We don't need a label for the legend. The text "Legend" shows up on my device and overlaps with the pie chart.